lms/djangoapps/branding/api.py

```diff
@@ -329,3 +329,39 @@ def _absolute_url_staticfile(is_secure, name):
     # For local development, the returned URL will be relative,
     # so we need to make it absolute.
     return _absolute_url(is_secure, url_path)
+
+
+def get_branding_links():
+    """
+    Return links customized according to site branding.
+    Returned dict will have the following key-value pairs
+        1. logo: logo url for the site
+        2. base_url: Base/Home URL of the site
+
+    Returned dict may have following key-value pairs depending upon misrosite configurations.
+         1.  terms_of_service_and_honor_code: Terms of service and honor code URL
+         2.  privacy_policy: Privacy Policy URL
+         3.  accessibility_policy: Accessibility Policy URL
+         4.  terms_of_service: Terms of Service URl
+         5.  honor_code: Honor Code URL
+         6.  about: URL of the about page
+         7.  blog: URL of site blog
+         8.  news: url of news page
+         9.  help-center: url of help center page
+         10. contact: url of contact us page
+         11. careers: url of careers page
+         12. donate: url of the page to accept donations
+         13. sitemap: url of the sitemap
+    """
+
+    legal_links = {item['name']: item['url'] for item in _footer_legal_links()}
+    footer_navigation_links = {item['name']: item['url'] for item in _footer_navigation_links()}
+
+    links = dict(
+        logo=get_logo_url(),
+        base_url=_absolute_url(True, ""),
+        **legal_links
```

Comment:
I'm not sure if we should pull `legal_links` and `footer_navigation_links` from branding app because if `ENABLE_MKTG_SITE` flag is off for a microsite it'll return either `#` or link from `settings.MKTG_URLS` which is not the required behavior.